packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -422,6 +428,12 @@ class SelectableText extends StatefulWidget {
   /// {@macro flutter.painting.textPainter.textWidthBasis}
   final TextWidthBasis textWidthBasis;
 
+  /// {@macro flutter.widgets.editableText.selectionWidthStyle}
```

Comment:
There is no corresponding template in editable text, you will need to create a template in there.